src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -53,23 +78,30 @@ class WorkDetail extends React.Component {
 
 WorkDetail.propTypes = {
   work: PropTypes.object,
```

Comment:
My esLint is getting mad at this line saying that `PropTypes.object` is disabled by default. I think it only raises a warning

Reply:
Yes, it objects to the object designation in multiple places.